admin/query/src/main/java/org/codice/ditto/replication/admin/query/sites/discover/GetReplicationSites.java

```diff
@@ -21,39 +21,54 @@
 import org.codice.ddf.admin.api.fields.FunctionField;
 import org.codice.ddf.admin.api.fields.ListField;
 import org.codice.ddf.admin.common.fields.base.BaseFunctionField;
+import org.codice.ddf.admin.common.fields.common.PidField;
+import org.codice.ddf.admin.common.report.message.DefaultMessages;
 import org.codice.ditto.replication.admin.query.ReplicationUtils;
 import org.codice.ditto.replication.admin.query.sites.fields.ReplicationSiteField;
+import org.codice.ditto.replication.api.data.ReplicationSite;
 
 public class GetReplicationSites extends BaseFunctionField<ListField<ReplicationSiteField>> {
 
   public static final String FIELD_NAME = "sites";
 
   public static final String DESCRIPTION =
-      "Retrieves all the sites currently configured for replication.";
+      "Retrieves all the sites currently configured for replication, or a single site identified by a pid.";
```

Comment:
✏️ Instead of `a single site ...`, maybe something like `a site given its identifier`.